internal/app/tfsec/adapter/aws/ecs/adapt.go

```diff
@@ -2,9 +2,91 @@ package ecs
 
 import (
 	"github.com/aquasecurity/defsec/provider/aws/ecs"
+	"github.com/aquasecurity/defsec/types"
 	"github.com/aquasecurity/tfsec/internal/app/tfsec/block"
 )
 
 func Adapt(modules []block.Module) ecs.ECS {
-	return ecs.ECS{}
+	return ecs.ECS{
+		Clusters:        adaptClusters(modules),
+		TaskDefinitions: adaptTaskDefinitions(modules),
+	}
+}
+
+func adaptClusters(modules []block.Module) []ecs.Cluster {
+	var clusters []ecs.Cluster
+	for _, module := range modules {
+		for _, resource := range module.GetResourcesByType("aws_ecs_cluster") {
+			clusters = append(clusters, adaptClusterResource(resource))
+		}
+	}
+	return clusters
+}
+
+func adaptClusterResource(resourceBlock block.Block) ecs.Cluster {
+	return ecs.Cluster{
+		Metadata: resourceBlock.Metadata(),
+		Settings: adaptClusterSettings(resourceBlock),
+	}
+}
+
+func adaptClusterSettings(resourceBlock block.Block) ecs.ClusterSettings {
+	if settingBlock := resourceBlock.GetBlock("setting"); settingBlock.IsNotNil() && settingBlock.GetAttribute("name").Equals("containerInsights") {
+		containerInsightsEnabled := settingBlock.GetAttribute("value").Equals("enabled")
+		return ecs.ClusterSettings{
+			ContainerInsightsEnabled: types.Bool(containerInsightsEnabled, settingBlock.Metadata()),
+		}
+	}
+
+	return ecs.ClusterSettings{
+		ContainerInsightsEnabled: types.Bool(false, resourceBlock.Metadata()),
```

Comment:
As this is not set, we treat it as the default value to show that it was unset in the check
```suggestion
		ContainerInsightsEnabled: types.BoolDefault(false, resourceBlock.Metadata()),
```

Reply:
(added new commit to address this, also marked the "thing" at line 90 as default too)